pages/operations/managing-applications/connect-application-db.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Connect to an Application DB
+description: How to connect to the underlying database for a deployed application.
+---
```

Comment:
should we give an example here and a description? like "In certain cases you'll want to access the database for an application directly. As an example, you may want to <insert some relevant reason why you'd access postgres>"

Reply:
This vastly varies from user to user, but I've added a sentence that addresses this.